test/achievement-specs.js

```diff
@@ -10,18 +10,54 @@ beforeEach(function syncDB () {
 describe('achievement', () => {
   context('GET /achievement', () => {
     it('should return achievements', async () => {
-      let a1 = await models.db.achievement.create({sequence: 1, name: 'London', distance: 100})
-      let a2 = await models.db.achievement.create({sequence: 2, name: 'Paris', distance: 200})
+      let a1 = await models.db.achievement.create({
+        sequence: 1,
+        name: 'London',
+        distance: 100,
+        description: 'AKF UK',
+        flag_name: 'greatbritain',
```

Comment:
I think that `GB` is better a value, but this is a test, so whatever.

Reply:
The flag_name value effectively is an icon. A corresponding filename is baked into the app. So renaming the attribute to icon_name.